examples/others/fragment/lv_example_fragment.h

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
The CI fails because this file is not included in `lv_example_others.h`

Reply:
I think it is properly included.